### ISSUE TITLE
Can't install Laravel-gamp package (Laravel 6.0) => PHP Fatal error

### ISSUE THREAD

**Issue by goodtrash** [irazasyed/laravel-gamp#25]
After running: 

`composer require irazasyed/laravel-gamp`

I get: 

`Loading composer repositories with package information`
`Updating dependencies (including require-dev)`
`Nothing to install or update`
`Generating optimized autoload files`

and then, the error: 

`PHP Fatal error:  Uncaught TypeError: Argument 1 passed to Composer\Autoload\ClassLoader::addClassMap() must be of the type array, int given, called in phar:///usr/local/bin/composer/src/Composer/Autoload/AutoloadGenerator.php on line 767 and defined in phar:///usr/local/bin/composer/vendor/composer/ClassLoader.php:92`

Do you know how can I solve this?

**Comment by goodtrash** [comment]
Hi.

I solved it. It was not related to composer.

My configuration is: Windows host, vagrant with Ubuntu and nfs sharing enabled. 

BTW, another and probably related issue, after running gulp three and more times to minimize the JavaScript sources, I got a 0 byte file as a result along with a NFS server error No such file or directory IO as mentioned here: winnfsd/vagrant-winnfsd#104 (comment) and only reloading vagrant & the NFS server solved it. Turned out to be an issue with Windows Defender, probably an exclusive file lock / race condition issue.

Excluding the whole synchronized folder from the Windows Defender "workspace" seems to do the trick, at least up till now it has – might work in this case for you too.

**Comment by goodtrash** [comment]
Hi.

I solved it. It was not related to composer.

My configuration is: Windows host, vagrant with Ubuntu and nfs sharing enabled.

BTW, another and probably related issue, after running gulp three and more times to minimize the JavaScript sources, I got a 0 byte file as a result along with a NFS server error No such file or directory IO as mentioned here: winnfsd/vagrant-winnfsd#104 (comment) and only reloading vagrant & the NFS server solved it. Turned out to be an issue with Windows Defender, probably an exclusive file lock / race condition issue.

Excluding the whole synchronized folder from the Windows Defender "workspace" seems to do the trick, at least up till now it has – might work in this case for you to

**Comment by goodtrash** [comment]
Hi.

I solved it. It was not related to composer.

My configuration is: Windows host, vagrant with Ubuntu and nfs sharing enabled.

I excluded my project folder from Windows Defender, restarted my computer, "vagrant up" my vagrant machine and it worked. 

For more detailed information, I found help in this link. Pay attention where it talks about the autoload_classmap.php file:

https://github.com/composer/composer/issues/5707
